free-1.md

```diff
@@ -45,6 +59,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
```

Comment:
Doing research into this is a good idea. I guess I was making the assumption Kuenning dealt with all this.